tests/admin_filters/tests.py

```diff
@@ -1352,6 +1377,15 @@ def test_simplelistfilter_with_queryset_based_lookups(self):
         self.assertEqual(choices[2]["query_string"], "?publication-decade=the+00s")
 
     def _test_facets(self, modeladmin, request, query_string=None):
+        self.djangonaut_book.employee = self.john
+        self.djangonaut_book.save()
+        self.django_book.employee = self.jack
+        self.django_book.save()
+        self.jack.department = self.dev
+        self.jack.save()
+        self.john.department = self.design
+        self.john.save()
+
```

Comment:
I think we can remove the data manipulation here (and below) and instead in `setUpTestData` move the department and employee creation above the book creation and have the employee assigned in the creation of djangonaut_book e.g.

```python
        cls.djangonaut_book = Book.objects.create(
            title="Djangonaut: an art of living",
            year=2009,
            author=cls.alfred,
            is_best_seller=True,
            date_registered=cls.today,
            availability=True,
            category="non-fiction",
            employee=cls.john,
        )
```

And then update the counts 👍